resources/public/api/conf/1.0/examples/register/register-max-post.json

```diff
@@ -90,14 +91,14 @@
       "line5": "ItsTheFinalLine"
     }
   },
-  "priorResidency": [
+  "applicantPriorResidency": [
     {
-      "priorStartDate": "10-10-2020",
-      "priorEndDate": "10-10-2020"
+      "startDate": "10-10-2020",
+      "endDate": "10-10-2020"
```

Comment:
This needs changing back to 'priorEndDate'

Reply:
done